packages/xo-web/src/xo-app/logs/backup-ng/log-alert-body.js

```diff
@@ -110,17 +111,58 @@ const TaskError = ({ task }) => {
   )
 }
 
-const Warnings = ({ warnings }) =>
-  warnings !== undefined ? (
+class Message extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      expanded: false,
+    }
+  }
+
+  render() {
+    const className = `text-${this.props.type} ${this.props.data ? 'message-expandable' : ''}`
+    const icon = this.props.type === 'warning' ? 'alarm' : 'info'
+
+    return (
+      <div>
+        <span className={className} onClick={() => this.setState({ expanded: !this.state.expanded })}>
+          <Icon icon={icon} /> {_(`backup:${this.props.message}`)}
```

Comment:
```suggestion
          <Icon icon={icon} /> {_(`backup:${this.props.message}`, this.props.data)}
```

and then you use dynamic message in translation like  `{displayed, number}x {icon} (of {total, number})`